Makefile

```diff
@@ -0,0 +1,11 @@
+GOROOTDIR := $(shell go env GOROOT)
+
+
+all: help
+help:
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
how about instead of printing the targets, we make the first target "wasm" and make it produce web/util.wasm and web/wasm_exec.js, and make the serve target depend on it? (cool one liner though)

Reply:
Way much better :D